--- preupg/utils.py.orig	2021-01-29 16:52:35.000000000 +0200
+++ preupg/utils.py	2023-12-20 10:22:28.516874749 +0200
@@ -380,14 +380,7 @@ class SystemIdentification(object):
     @staticmethod
     def get_variant():
         """Function return a variant"""
-        redhat_release = FileHelper.get_file_content("/etc/redhat-release", "rb")
-        if redhat_release.startswith('Fedora'):
-            return None
-        try:
-            rel = redhat_release.split()
-            return rel[4]
-        except IndexError:
-            return None
+        return "Server"
 
 
 class TarballHelper(object):
